src/tests/ftest/container/snapshot.py

```diff
@@ -327,72 +357,70 @@ def test_snapshots(self):
                    epoch.
                    ==>Repeat step(1) to step(4) for multiple snapshot tests.
                 (5)Verify the snapshots data.
-                (6)Destroy the snapshot.
+                (6)Destroy the snapshot individually.
                 (7)Check if still able to Open the destroyed snapshot and
                    Verify the snapshot removed from the snapshot list.
+                (8)Destroy the container snapshot.
         Use Cases: Require 1 client and 1 server to run snapshot test.
                    1 pool and 1 container is used, num_of_snapshot defined
                    in the snapshot.yaml will be performed and verified.
-        :avocado: tags=all,small,smoke,pr,snap,snapshots
+        :avocado: tags=all,small,smoke,snap,snapshots,full_regression
```

Comment:
As this is being removed from 'pr' tag can you please make sure this test runs in CI before it is merged.
You need to add "Test-tag: pr snapshots" in your last commit before pushing.

Reply:
will do.